core/application/bloodDonationWorkflow/BloodDonationMessages.ts

```diff
@@ -0,0 +1,8 @@
+import { BLOOD_REQUEST_MESSAGE_TEMPLATE } from '../utils/messageConstants'
+import { replaceTemplatePlaceholders } from '../utils/formatString'
+import { UrgencyLevel } from '../../../commons/dto/DonationDTO'
+
+export function getBloodRequestMessage(urgencyLevel: string, bloodGroup: string, shortDescription: string): string {
+  const urgency = urgencyLevel === UrgencyLevel.URGENT ? 'Urgent' : ''
```

Comment:
empty string?
should be instead regular?

Reply:
The message would look wired.  like  `Raguler A+ blood needed`

Reply:
erm ok